appcenter-cli-1ES.yml

```diff
@@ -0,0 +1,135 @@
+          inputs:
+            versionSpec: '14.x'
+          displayName: 'Install Node.js'
+
+        - script: |
+            npm ci
+            npm run build
+          displayName: 'Install and build'
+
+        - script: |
+            npm run test:ci || [ -f './test-results.xml' ]
+          displayName: 'Run CLI tests (with JUnit output)'
+
+        - task: PublishTestResults@2
+          inputs:
+            testResultsFormat: 'JUnit'
+            testResultsFiles: 'test-results.xml'
+            searchFolder: '$(Build.SourcesDirectory)'
+            failTaskOnFailedTests: true
+            testRunTitle: 'CLI unit tests'
+          displayName: 'Verify CLI unit test result'
+
+        - script: |
+            npm ci
+            npm run build
+          displayName: 'Install and build appcenter-file-upload-client-node'
+          workingDirectory: $(Build.SourcesDirectory)/appcenter-file-upload-client-node
+
+        - script: |
+            npm run test:ci || [ -f './test-results.xml' ]
+          displayName: 'Run appcenter-file-upload-client-node tests'
+          workingDirectory: $(Build.SourcesDirectory)/appcenter-file-upload-client-node
+
+        - task: PublishTestResults@2
+          inputs:
+            testResultsFormat: 'JUnit'
+            testResultsFiles: 'test-results.xml'
+            searchFolder: '$(Build.SourcesDirectory)/appcenter-file-upload-client-node'
+            failTaskOnFailedTests: true
+            testRunTitle: 'appcenter-file-upload-client-node unit tests'
+          displayName: 'Verify appcenter-file-upload-client-node unit test result'
+
+        - script: |
+            npm pack
+            npm install -g appcenter-cli*.tgz
+          displayName: 'Package appcenter-cli'
+          workingDirectory: $(Build.SourcesDirectory)
+
+        - task: AzureKeyVault@1
+          inputs:
+            azureSubscription: '$(API_TOKEN_KEY_VAULT_AZURE_SUBSCRIPTION)'
+            KeyVaultName: '$(API_TOKEN_KEY_VAULT)'
+            SecretsFilter: 'ac-cli-account-api-token'
+
+        - task: PowerShell@2
+          inputs:
+            filePath: '$(Build.SourcesDirectory)/test/functional/Invoke-FunctionalTests.ps1'
+            arguments: '-Token $(ac-cli-account-api-token)'
+            pwsh: true
+            workingDirectory: $(Build.SourcesDirectory)
+          displayName: 'Run functional tests'
+
+        - script: |
+            [ -f './test/functional/testresult.xml' ]
+          displayName: 'Verify that functional test result file exists'
+          workingDirectory: $(Build.SourcesDirectory)
+
+        - task: PublishTestResults@2
+          inputs:
+            testResultsFormat: 'NUnit'
+            testResultsFiles: 'testresult.xml'
+            searchFolder: '$(Build.SourcesDirectory)/test/functional'
+            failTaskOnFailedTests: true
+            testRunTitle: 'Functional tests'
+          displayName: 'Verify functional test result'
+
+        - script: |
+            npm pack
```

Comment:
We alreade created pack on line 84 - how is this different?

Reply:
Removed.